python/lsst/ip/diffim/diaCatalogSourceSelector.py

```diff
@@ -201,43 +201,56 @@ def selectStars(self, exposure, sourceCat, matches=None):
         #
         starCat = SourceCatalog(sourceCat.schema)
 
+        if display:
```

Comment:
From below it looks to me that `if display and displayExposure:` would be enough?